pkg/apis/policy/v1beta1/defaults.go

```diff
@@ -32,4 +32,8 @@ func SetDefaults_PodSecurityPolicySpec(obj *policyv1beta1.PodSecurityPolicySpec)
 		t := true
 		obj.AllowPrivilegeEscalation = &t
 	}
+	if obj.RunAsGroup == nil {
+		runAsGroupOpts := policyv1beta1.RunAsGroupStrategyOptions{Rule: policyv1beta1.RunAsGroupStrategyRunAsAny}
```

Comment:
no defaulting on auto-cleared alpha fields. this causes PSP objects read from the API to contain this field, which we don't want when the alpha feature is disabled